### PR TITLE
175652133 dev readme

### PR DESCRIPTION
**Pull request by corydodt**

### CODE REVIEW

**Comment by corydodt** [comment]
It's been challenging to run an instance of jobbergate-api, and it should be possible to run the service by following the README.

This provides a new tool `scripts/serverlessenv.py` to help you configure your local environment, and updates the README with current procedures for spinning up a dev env and using it with django tools.

Most changes are in:
- scripts/serverlessenv.py: new script, supersedes export_vars.sh and makes it possible to run either localhost or serverless-deploy easily.
- jobbergate_api/settings.py: reorganized to make it easier to run localhost/serverless-deploy from the same settings file.
- README: fixed instructions, minor reorganization

**Comment by corydodt** [comment]
Here's a direct link to the formatted version of the README: https://github.com/omnivector-solutions/jobbergate-api/blob/master/README.md

**Review by ducks23 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,55 +1,117 @@
...
+
+    - [Docker for Mac](https://docs.docker.com/docker-for-mac/install/)
+
+1. Using serverless + cloudformation, deploy your instance to AWS:
+
+    ```bash
+    npx serverless deploy --stage your_name
+    ```
+
+    **IMPORTANT**: You must visit
```
Comment (jamesbeedy) [inline README.md:61]: thank you for adding this bit about the manual validation - 💯

File: README.md [hunk 2]
```diff
@@ -1,55 +1,117 @@
...
-# Make migrations
-./manage.py makemigrations
+# Note: make sure you have successfully used serverless deploy, above, or this will fail
 
-# Migrate schema
-./manage.py migrate
 
-# Collect static files
+# Set REQUIRED env vars, for this shell session
+eval $(./scripts/serverlessenv.py your_name)
```
Comment (jamesbeedy) [inline README.md:85]: Excellent rework of what was!

**Review by jamesbeedy — APPROVED** [review]
+1 to these changes

I <3 the `serverlessenv` implementation and the readme rewrite! Nice work!

**Pull request merged**